### PR TITLE
[soltest] Add support for string literals

### PR DESCRIPTION
**Pull request by erak**

### CODE REVIEW

**Review by chriseth — CHANGES_REQUESTED** [review]
Please add a test where two strings are returned and also `(string, uint, string)`

File: test/libsolidity/semanticTests/strings.sol [hunk 1]
```diff
@@ -0,0 +1,8 @@
+contract C {
+    string public shortString = "any";
+    string public longString = "anyanyanyanyanyanyanyanyanyanyanyanyanyanyanyany";
+}
+// ====
+// EVMVersion: >=constantinople
```
Comment (chriseth) [inline test/libsolidity/semanticTests/strings.sol]: Why constantinople?

Comment (erak) [reply test/libsolidity/semanticTests/strings.sol]: This will be removed. The test will be part of the smoke tests.

**Comment by erak** [comment]
@chriseth @Marenz Updated. It seems that the type detection based on the ABI during the update process needs some special treatment. But since this PR is blocking two others, it might make sense to do that in a follow-up PR. But I could also fix it with this PR.

**Review by chriseth — APPROVED** [review]

**Pull request merged**